### PR TITLE
Brian's version

### PR DESCRIPTION
**Pull request by fafcrumb**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (robertavram) [inline LICENSE]: keep the apache license remove the bsd

File: app/elements/agreement-data/agreement-data.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Polymer({
+      is: 'agreement-data',
+      observers: ['_updateAgreements(data.*)'],
+      properties: {
+        data: {
+          type: Array,
+          notify: true,
+          value: []
+        },
+        url: {
```
Comment (robertavram) [inline app/elements/agreement-data/agreement-data.html]: I believe you can remove url from here

File: app/elements/agreement-data/agreement-data.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../common/etools-ajax/etools-ajax.html">
+
+<dom-module id="agreement-data">
+  <template>
+    <etools-ajax
+          auto
+          id="ajax"
+          url="[[url]]"
```
Comment (robertavram) [inline app/elements/agreement-data/agreement-data.html]: you can remove url from here as well.. I think

File: app/elements/behaviors/common/utils.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    // transform object to array
+    return Object.keys(obj).map(function(key) {
+        return {
+            name: key,
+            value: obj[key]
+        };
+    });
+  }
+};
+var config = {
```
Comment (robertavram) [inline app/elements/behaviors/common/utils.html]: remove all endpoints not used in this portal from config

File: app/elements/common/etools-localstorage/etools-localstorage.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
+
```
Comment (robertavram) [inline app/elements/common/etools-localstorage/etools-localstorage.html]: remove this element for now

File: app/elements/partner-api.html [hunk 6]
```diff
@@ -1,205 +0,0 @@
...
-      },
-      db: {
-        type: Object
-      },
-      dbView: {
-        type: Object
-      }
-    },
-    ready: function() {
-      var db = new loki('etools');
```
Comment (robertavram) [inline app/elements/partner-api.html]: lokijs is worth looking into.. this sorting mechanism rocks

Comment (fafcrumb) [inline app/elements/partner-api.html]: It looks pretty awesome. Should I use this for storing (and then searching/filtering) the table data?

**Comment by fafcrumb** [comment]
Closing because changes are redundant and appear in #4

**Pull request closed without merging**